backend/Dockerfile

```diff
@@ -1,4 +1,40 @@
-FROM l.gcr.io/google/bazel:0.24.0 as builder
+FROM ubuntu:18.04 as builder
```

Comment:
Can you please split out this part into another Dcokerfile that creates and image that's analogous to `l.gcr.io/google/bazel`, but works on ARM64?

Maybe that Dockerfile can later be added to the Bazel project.

Reply:
Thanks for your comment, and I think yes, however can you please be more specific? Shall I upload another Dockerfile named "Dockerfile.aarch64" to build the backend and leave the existing Dockerfile for backend as something like "Dockerfile.x86_64", or just split the bazel part as "Dockerfile.bazel"? 

Thanks!

Reply:
It would be great if you create `Dockerfile.bazel`.
Maybe you can then select the base image based on Docker build argument or some other mechanism.

Reply:
Hi @Ark-kun !

Referencing [comment#1 ](https://github.com/moby/moby/issues/3378#issuecomment-31325743) and [comment#2 ](https://github.com/moby/moby/issues/3378#issuecomment-31326051)in [moby#3378](https://github.com/moby/moby/issues/3378), I don't think we have a official way to achieve this for now, unless google releases the multiarch version of bazel image, but I did refine the Dockerfile for pipeline backend to make it a little bit lighter. Since the builder image will not be used as the final image of pipeline backend, I think the method for now would also work fine. The bazel installation commands in the updated `backend/Dockerfile` can serve as the `Dockerfile.bazel` in the future.

Reply:
Is it possible to solve this using the following Dockerfile?
```
ARG BAZEL_IMAGE=l.gcr.io/google/bazel:0.24.0 
FROM $BAZEL_IMAGE as builder
...
```

Reply:
Ahhh cool @Ark-kun ! Thanks very much for this awesome advice. I have pushed a new commit by following this, please review.